src/sage/algebras/cluster_algebra.py

```diff
@@ -462,7 +462,7 @@ def d_vector(self) -> tuple:
             sage: x.d_vector()
             (1, 1, 2, -2)
         """
-        monomials = self.lift().dict()
+        monomials = self.lift().monomial_coefficients()
```

Comment:
I'm not sure whether I should replace this with
```suggestion
        monomials = self.monomial_coefficients()
```
throughout.

Reply:
I would not change this. In general, ambient spaces has a very different basis than sub/quotient spaces, and `monomial_coefficients` should return the `dict` with respect to the basis of the module (algebra in this case). So this could lead to a change in behavior. At the very least, the meaning of what the code is doing has changed (even if the output is the same).